test/suites/auth.sh

```diff
@@ -1240,9 +1240,8 @@ entities_enrichment_with_entitlements() {
   lxc auth identity-provider-group delete test-idp-group3
 
   # Image
-  lxc init images:busybox/1.36.1 c1
-  lxc delete c1
-  imgFingerprint=$(lxc image list --format json | jq -r '.[] | select(.update_source.alias == "busybox/1.36.1") | .fingerprint')
+  ensure_import_testimage
+  imgFingerprint="$(lxc query /1.0/images/aliases/testimage | jq -r .target)"
```

Comment:
```suggestion
  imgFingerprint="$(lxc image info testimage| awk '/^Fingerprint/ {print $2}')"
```
Looks like the `.target` was a typo/copy-n-pasto?